angular-client/src/components/battery/battery.component.css

```diff
@@ -0,0 +1,16 @@
+.battery {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.battery-nub {
+  width: 14px;
+  height: 7px;
+  background-color: #efefed;
+}
+
+.battery-bar {
+  width: 30px;
```

Comment:
these should be configurable, shouldnt be a fixed size might want bigger or smaller batteries. when creating components think about how the user would use the component and the different ways they would want to use it.